syncd build failure on broadcom.

On master branch, in local test I have met with one below issue and not able to advance submodule sonic-sairedis. It causes the failure of syncd build for platform broadcom.
 
configure: libsai found
checking for swig3.0... /usr/bin/swig3.0
checking whether CXX supports -Wcast-align=strict... yes
checking whether CXX supports -Wno-cast-function-type... yes
checking SAI headers API version and library version check... failed
configure: error: "SAI headers API version and library version mismatch"
make[1]: *** [Makefile:405: config.status] Error 1
make[1]: Leaving directory '/sonic/src/sonic-sairedis'
make: *** [Makefile:594: ../aclocal.m4] Error 2
 
junhuazhai@e8ea45ebfbd4:/sonic/src/sonic-sairedis/syncd$ dpkg -l | grep saibcm
ii  libsaibcm                              7.1.0.0-1                               amd64        Switch Abstraction Interface sdk based on Broadcom SAI
ii  libsaibcm-dev                          7.1.0.0-1                               amd64        development files for Switch Abstraction Interface sdk
 
 
This issue is due to [[configure.ac] implement SAI API version check by #1000  The API sai_query. _api_version is new and has not been implemented in latest BRCM SAI. This PR only has the API implementation for vs SAI lib. Not sure whether the API has been included in other vendor SAI libs.

--- a/configure.ac
+++ b/configure.ac
@@ -186,6 +186,7 @@ AM_COND_IF([SYNCD], [
 AM_COND_IF([SAIVS], [], [
 SAVED_FLAGS="$CXXFLAGS"
 CXXFLAGS="-lsai -I$srcdir/SAI/inc -I$srcdir/SAI/experimental -I$srcdir/SAI/meta"
+AC_CHECK_FUNC(sai_query_api_version, [
 AC_MSG_CHECKING([SAI headers API version and library version check])
 AC_TRY_RUN([
 extern "C" {
@@ -201,7 +202,7 @@ int main() {
 }],
 [AC_MSG_RESULT(ok)],
 [AC_MSG_RESULT(failed)
-AC_MSG_ERROR("SAI headers API version and library version mismatch")])
+AC_MSG_ERROR("SAI headers API version and library version mismatch")])])
 CXXFLAGS="$SAVED_FLAGS"
 ])])
 

